public/uploadStudentList.php

```diff
@@ -1,53 +1,45 @@
 
-<?php 
+<?php
 require_once "../modules/models/user.php";
 require_once "../common.php";
 
     $create_success = false;
 
-    if (isset($_POST['submit2'])) 
-	{
-        
-		$students = fopen($_POST["fileID"], "r");
+    if (isset($_POST['submit2'])) {
+        $students = fopen($_POST["fileID"], "r");
         $headers = fgetcsv($students, 1000, ",");
         $count = 0;
-		while (($studentData = fgetcsv($students, 1000, ",")) !== FALSE) 
-        {
-                $user = new User();
-                $user->student_id = $studentData[0];
-                $user->first_name = $studentData[1];
-                $user->last_name = $studentData[2];
-                $user->email = $studentData[3];
-                $user->is_admin = 0;
-                $user->is_instructor = 0;
-                $user->password_digest = password_hash("welcome", PASSWORD_DEFAULT);
-    
-                try {
-                    $user->save();
-                    $count++;
-                    $create_success = true;
-                }  catch(PDOException $error) {
-                     echo "<br>" . $error->getMessage();
-                }
-            
-    
+        while (($studentData = fgetcsv($students, 1000, ",")) !== false) {
+            $user = new User();
+            $user->student_id = $studentData[0];
+            $user->first_name = $studentData[1];
```

Comment:
I have been running a linter locally, and I recommend you guys to do the same :) it helps a lot with formatting (php code is super messy as it is so I find this very handy)

https://github.com/FriendsOfPHP/PHP-CS-Fixer heres what I use